### PR TITLE
Code clean up to remove some warnings

### PR DESCRIPTION
**Pull request by kevinsi4508**

### CODE REVIEW

File: bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/HadoopSerializationCoder.java [hunk 1]
```diff
@@ -70,15 +69,6 @@ private void readObject(java.io.ObjectInputStream in) throws IOException, ClassN
...
 
-  /**
-   * Disable backwards compatibility with previous versions that were serialized
-   * @throws InvalidObjectException
-   */
-  private void readObjectNoData() throws InvalidObjectException {
-    throw new InvalidObjectException("Stream data required");
-  }
-
-
```
Comment (igorbernstein2) [inline bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/HadoopSerializationCoder.java]: why remove this?

Comment (kevinsi4508) [reply bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/HadoopSerializationCoder.java]: I thought it is not used, no?

Comment (igorbernstein2) [reply bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/HadoopSerializationCoder.java]: It's implicitly used for java serialization:
https://docs.oracle.com/javase/7/docs/platform/serialization/spec/input.html#6053

Comment (kevinsi4508) [reply bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/HadoopSerializationCoder.java]: Good catch! I was having doubt about this one. I should have pointed out earlier.
I added some comment to remind us not to delete this method.

**Comment by kevinsi4508** [comment]
ping

**Pull request merged**